test/Tester/CollectionFixtures.cs

```diff
@@ -17,5 +23,21 @@ protected override void CheckPreconditionsOrThrow()
             base.CheckPreconditionsOrThrow();
             TestUtils.CheckForAzureStorage();
         }
+
+        public class SiloBuilderFactory : ISiloBuilderFactory
+        {
+            public ISiloHostBuilder CreateSiloBuilder(string siloName, ClusterConfiguration clusterConfiguration)
+            {
+                return new SiloHostBuilder()
+                    .ConfigureSiloName(siloName)
+                    .UseConfiguration(clusterConfiguration)
+                    .UseAzureTableMembership(options =>
```

Comment:
hmmm, not sure about defining this at this level. The `BaseAzureTestClusterFixture` is mostly used for tests that require Azure for storage and/or membership, but most often than not they don't require membership. I would define this silo builder factory for the few specific tests that do require changing membership. At the very least, define this factory as a standalone thing outside the `BaseAzureTestClusterFixture` type, and with a clear name of what is doing, so that you can reuse it in the tests that actually do require a different membership